extensions/amp-analytics/0.1/test/test-amp-analytics.js

```diff
@@ -33,6 +33,8 @@ import {
   installUrlReplacementsService,
 } from '../../../../src/service/url-replacements-impl';
 import * as sinon from 'sinon';
+import {installStorageService} from '../../../../src/service/storage-impl';
+import '../../../amp-analytics/0.1/amp-analytics';
```

Comment:
why did moving storage service to runtime, caused this file to now require importing amp-analytics?